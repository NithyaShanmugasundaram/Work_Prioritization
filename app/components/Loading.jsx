import Spinner from "react-bootstrap/Spinner";
export default function Loading() {
  return (
    <div className="mt-10 text-center">
      <Spinner animation="border" variant="success" />
      <p>Hopefully not for to long :)</p>
    </div>
  );
}
