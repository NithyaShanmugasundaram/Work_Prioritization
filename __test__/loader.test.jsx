import React from 'react';
import renderer from 'react-test-renderer';
import Loading from './../app/components/Loading';

test('MyComponent snapshot', () => {
    const component = renderer.create(<Loading />);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });