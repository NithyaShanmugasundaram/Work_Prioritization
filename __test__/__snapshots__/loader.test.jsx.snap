// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MyComponent snapshot 1`] = `
<div
  className="mt-10 text-center"
>
  <div
    className="spinner-border text-success"
  />
  <p>
    Hopefully not for to long :)
  </p>
</div>
`;
